crates/node-core/src/args/utils.rs

```diff
@@ -77,6 +78,8 @@ pub fn genesis_value_parser(s: &str) -> eyre::Result<Arc<ChainSpec>, eyre::Error
         "holesky" => HOLESKY.clone(),
         #[cfg(not(feature = "optimism"))]
         "dev" => DEV.clone(),
+        #[cfg(not(feature = "optimism"))]
```

Comment:
why?

Reply:
The bitfinity is only supported in non-optimism feature; the reth node can run on optimism(rollup node)